src/web/screens/sharingPermissions.tsx

```diff
@@ -127,7 +128,7 @@ function SharingPermissions() {
         <br />
         Note: This only enables the sharing permission. No data is sent.
       </p>
-      <div className='bulk-add-and-search-collapsibles'>
+      <ContentContainer>
```

Comment:
One reason I think this is better is because if we were add another component at the bottom of this div, we would need to change the above `className` to something like `x-and-y-and-z`